MapboxNavigation.podspec

```diff
@@ -36,7 +36,7 @@ Pod::Spec.new do |s|
 
   # ――― Resources ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
-  s.resources = ['Sources/MapboxNavigation/Resources/*/*', 'Sources/MapboxNavigation/Resources/*']
+  s.resources = ['Sources/MapboxNavigation/Resources/*/*', 'Sources/MapboxNavigation/Resources/*', 'Sources/MapboxNavigation/MBXInfo.plist']
```

Comment:
We can consider moving `MBXInfo.plist` to `Resource` directory, shouldn't break anything.

Reply:
Done, changed Package.swift as well